src/org/thoughtcrime/securesms/recipients/Recipient.java

```diff
@@ -264,6 +267,8 @@ public void setContactUri(@Nullable Uri contactUri) {
       return Util.join(names, ", ");
     }
 
+    if (this.nickname != null) return this.nickname;
```

Comment:
The pattern in this project has generally been to write the block `if () {...` so I think that should be followed